administrator/components/com_fields/src/Model/FieldsModel.php

```diff
@@ -168,14 +168,14 @@ protected function getListQuery()
 		$query->select('ua.name AS author_name')->join('LEFT', '#__users AS ua ON ua.id = a.created_user_id');
 
 		// Join over the field groups.
-		$query->select('g.title AS group_title, g.access as group_access, g.state AS group_state, g.note as group_note');
+		$query->select('g.title AS group_title, g.access as group_access, g.state AS group_state, g.note as group_note, g.params as group_params');
 		$query->join('LEFT', '#__fields_groups AS g ON g.id = a.group_id');
 
 		// Filter by context
 		if ($context = $this->getState('filter.context'))
 		{
 			$query->where($db->quoteName('a.context') . ' = :context')
-				->bind(':context', $context);
+			->bind(':context', $context);
```

Comment:
Please revert this unrelated change since it violates code style rules.

Reply:
```suggestion
				->bind(':context', $context);
```
Since this line belongs to the previous line, i.e. they are one statement, this line needs to be intended by one tab more than the previous one.